.github/workflows/comment_commands.yml

```diff
@@ -0,0 +1,164 @@
+name: Comment Commands to Trigger CI
+on:
+  issue_comment:
+    types: created
+
+permissions:
+  checks: write
+
+jobs:
+  pandas_nightly:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: (github.event.issue.pull_request) && github.event.comment.body == '/pandas_nightly'
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install project dependencies
+        uses: ./.github/setup
+        with:
+          os: ubuntu-latest
+          python-version: "3.11"
+
+      - name: Run pytest (against pandas nightly)
+        id: tests-step
+        run: poetry run poe pytest --nightly
+
+      - name: Get head sha and store value
```

Comment:
Can you please add a few comments here in the file and explain what these steps do and why we need them (why do we need to get the hash)?

I'm not very familiar with the github CI scripts.

Reply:
Yes all will be documented to reflect what we are doing, agree it is not always super clear.